hack/local-up-cluster.sh

```diff
@@ -680,8 +677,13 @@ EOF
 fi
 }
 
+# validate that etcd is: not running, in path, and has minimum required version.
+kube::etcd::validate
+
 if [[ "${CONTAINER_RUNTIME}" == "docker" ]]; then
-  test_docker
+  # source to get common docker test
+  #source "${KUBE_ROOT}/build-tools/common.sh"
+  ${KUBE_ROOT}/build-tools/common.sh kube::build::ensure_docker_daemon_connectivity
```

Comment:
1. yeah saw that happened recently
2. yep it works, it's how you call a function when you don't want to source it, tried pulling from source but it got complicated because of the environment variables called in that `common.sh` script.
3. I am guessing you mean /hack/lib/init.sh? if so yeah they both source this file and would be a good place assuming that the function can cleanly be moved. Ill try it out.